soc/intel/intel_niosv/niosv/Kconfig.defconfig.series

```diff
@@ -1,10 +1,11 @@
-# Copyright (C) 2023, Intel Corporation
+# Copyright (C) 2023-2024, Intel Corporation
+#
 # SPDX-License-Identifier: Apache-2.0
 
 if SOC_SERIES_NIOSV
 
-config SOC_SERIES
-	default "niosv"
+config ARCH
+	default "riscv"
```

Comment:
needs this for HWMv2 build with correct `arch/riscv` directory overriding the value from here

https://github.com/zephyrproject-rtos/zephyr/blob/592a7cd6a750bdc715f07f004a25226618a89012/arch/riscv/Kconfig#L7-L10

@tejlmand is the above the expected way for HWMv2 to deal with ARCH, `riscv` in particular ?

Reply:
Haven't reviewed this but riscv porting is not working at present due to differences in name of arch and folders, need to cherry-pick some changes over

Reply:
> Haven't reviewed this but riscv porting is not working at present due to differences in name of arch and folders, need to cherry-pick some changes over

it should be this Kconfig change to allow riscv build, at least it is for intel_niosv

Reply:
https://github.com/zephyrproject-rtos/zephyr/pull/68853 is the fix